src/components/RoomView/Card.tsx

```diff
@@ -55,11 +55,10 @@ const Card = (word: CardObj) => {
         updates[`${newHistoryKey}`] = newGameHistory;
         update(gameHistoryRef, updates);
         update(singleCardRef, { isVisibleToAll: true, teamId: cardBelongsTo });
-        /* below sets the cards to 0 and declares opposing team as winner in 
-        roomview seems dishonest bc they dont actually have 0 remaining cards, 
-        but it'll  trigger code that is doing what we want it to in roomview 
-        instead of writing redundant logic*/
-        update(gameRef, { gameStatus: 'complete', team2RemainingCards: 0 });
+        update(gameRef, { gameStatus: 'complete'});
```

Comment:
this is no longer dishonest! when an assassin is clicked, the winners and losers are set respectively and the actions to redux in roomview when game is 'complete' are triggered